lib/phases/phase.js

```diff
@@ -505,15 +505,17 @@ Phase.prototype.calculateStats = function() {
         .forEach(function(key) {
           var values = contextResults[key];
           var mean = stats.mean(values);
+          var stdev = stats.stdev(values);
 
           statistics[contextKey].push({
             Metric: key,
             Mean: mean,
             Median: stats.median(values),
             Min: Math.min.apply(Math, values),
+            '95th Perc.': stats.percentile(values, 0.95) || mean,
```

Comment:
Wouldn't that be easily confused with the 95% confidence interval? How about we just remove this altogether?